src/include/verbose.h

```diff
@@ -134,6 +134,20 @@ struct __wt_verbose_multi_category {
 #define WT_VERBOSE_ISSET(session, category) \
     WT_VERBOSE_LEVEL_ISSET(session, category, WT_VERBOSE_LEVEL_DEFAULT)
 
+/* Set the verbose level and save the previous value. */
+#define WT_VERBOSE_SAVE(session, verbose_tmp, category, level)   \
```

Comment:
One more small tweak. Could we call this `WT_VERBOSE_SET_AND_SAVE` to make it clear we're also modifying the verbose level?

Reply:
hi, @ajmorton 
Thank you for your patient guidance, I have change the code again.

thanks.